package.json

```diff
@@ -24,9 +24,9 @@
     "gatsby-image": "^2.2.43",
     "gatsby-plugin-mdx": "^1.0.82",
     "gatsby-plugin-react-helmet": "^3.2.0",
-    "gatsby-plugin-sharp": "^2.4.12",
+    "gatsby-plugin-sharp": "^2.5.3",
     "gatsby-source-filesystem": "^2.1.55",
-    "gatsby-transformer-sharp": "^2.3.18",
+    "gatsby-transformer-sharp": "^2.4.3",
```

Comment:
Do you mind reverting the package.json (and subsequent package-lock.json) changes so this PR just merges in your content?